src/constant/symbols.js

```diff
@@ -0,0 +1,25 @@
+export const GAME_SYMBOL = {
+  ERROR: '[ERROR]',
+  RESTART: 'y',
+  EXIT: 'n',
+  INPUT: '>',
+  DASH: '-',
+};
+
+export const PURCHASE_SYMBOL = {
+  UNIT: 1000,
+  RANGE_MIN: 1000,
+};
+
+export const LOTTO_SYMBOL = {
+  COUNT: 6,
+  RANGE_MIN: 1,
+  RANGE_MAX: 45,
+  PRIZE: {
+    1: 2000000000,
```

Comment:
큰 숫자는 읽기가 어려우니 numeric literal을 사용해보셔도 좋을거 같아요!
```suggestion
    1: 2_000_000_000,
```

Reply:
안 그래도 0이 몇 개인지 여러 번 헷갈렸었는데 .. 좋은 방법 감사합니다!